.idea/rebel_project.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="JRebel Project data">
+    <option name="privateKey" value="3082025d02010002818100e85c3db222a65924a8bde52fa275fd4c7f3f1a9b013515ea1cda990aa2d0cd085cc57ccafcab8d0b90fdfd49457a2e566e226d5986d32133c36ab7f0043c554578cb1d5a26a23714d0c66dab11d927e46a5cf5c6ab0210115bab49a829e55614812d1a3a97f4c23596b125a8d3b66fd6b2da1b804001f18d3a8b053c7d20cc6902030100010281800c06ba2abe205dfcc7bf0c5daacaf0914504bf1bc6b438ae25502b1ba1d1accb020236b34ae0962c4314e24728916a548edeb94a28f04487a9559caf8f888fe12f5f42952b772bfe7ee305f356a065be18ea8d2e865791408a29926073d1ee4fc83bdec47c4d40851e36cca5a8d56f6aad53132b97affd062b2eac9cb1f0b401024100f7f31082dfc521a46b416c1ebf88406966da4b12ea149582a1ae888bfca25b91798f5976ce36a7f5afcc40bfc03d4e321fe2597c3ec957b250ca7adc4879dd2d024100efe799ca241eef93cd4104b0ce695445fb9cc5ca6e5acad9f0c577e693b92844f36a81f20c06cb94869934e035a79a639ccc3fb37cb973f4dc1a84b00356c9ad02410099b5c162a0be60a0c12d098df646cd478a4f8182f442567d6925fdd86a816c2abf2532dd07f1b2260919a27a3cfec45f268cf7d4a247c769efd3d65e813faaf1024100defe23082edd068a16ee89a2a2aa6b2805c1ea63c881e22e044c38affdbdccbd9298fca2752557d3fdeca8a68398ffbbf9c22caff48e19478fd3a2644d263d9d0240717272fcc904c871da8a06c29eb5a4c9268c0e8d972ea66f6b308ae9403228c813fe1ab5dfdf11edb8385d181b7bb586cca39bec2ab57b4bbd7fed66287d6982" />
```

Comment:
It's not my private key, it gets auto generated by JRebel every time IntelliJ does a full project build.